RecoTauTag/HLTProducers/src/L2TauJetsMerger.cc

```diff
@@ -58,8 +58,8 @@ void L2TauJetsMerger::produce(edm::StreamID iSId, edm::Event& iEvent, const edm:
     tauL2jets->push_back(myTmpJets[0]);
     CaloJetCollection tmp;
     for (unsigned int i = 1; i < myTmpJets.size(); ++i) {
-      double DR = ROOT::Math::VectorUtil::DeltaR(myTmpJets[0].p4(), myTmpJets[i].p4());
-      if (DR > 0.1)
+      double DR2 = ROOT::Math::VectorUtil::DeltaR2(myTmpJets[0].p4(), myTmpJets[i].p4());
+      if (DR2 > 0.1 * 0.1)
```

Comment:
```suggestion
      if (DR2 > 0.01)
```

Reply:
No, I prefer the cleaner expression of mine. For the compiler they'll be the same anyhow